features/languages.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Languages
+@QA @Languages
+Scenario: Ensure you can change languages in CommCare
+	Then I install the ccz app at "languages.ccz"
+	Then I login with username "clark" and password "123"
+
+	#Check that default language deployed
+	Then I press start
+	Then I select module "Basic Form Test"
+	And I select form "Languages"
+
+	Then I see the text "Enter a name:"
+	Then I enter text "English"
+	Then Submit

```

Comment:
You don't need to submit the form. Doing so causes the test to fail sometimes because calabash must wait for the submission to go through, which takes variable time. 
You can remove this line and leave the `I go back to the home screen` line